cmd/minishift/cmd/start.go

```diff
@@ -302,13 +317,12 @@ func initSubscriptionManagerFlags() {
 }
 
 // clusterUp downloads and installs the oc binary in order to run 'cluster up'
-func clusterUp(config *cluster.MachineConfig) {
-	oc := cache.Oc{
+func clusterUp(config *cluster.MachineConfig) (*cache.Oc, error) {
```

Comment:
@hferentschik , yes I am thinking of that too. `clusterUp` will just prepare the command and run it.